social-network/src/ArticleCard.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Image from 'react-bootstrap/Image';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBookmark as farBookmark } from '@fortawesome/free-regular-svg-icons'
+import { faStar as fasStar } from '@fortawesome/free-solid-svg-icons'
+import { faVolumeUp as fasVolumeUp } from '@fortawesome/free-solid-svg-icons'
+
+import PropTypes from 'prop-types';
+
+export default class ArticleCard extends React.Component {
+    static propTypes = {
+        article : PropTypes.object,
```

Comment:
This works, though in practice you will often be using shape to add more details to the expected keys in the object.